docs/source/examples/spot-jobs.rst

```diff
@@ -6,57 +6,88 @@ Managed Spot Jobs
 
-(2) **Saving and loading checkpoints**: (For ML jobs) Application code should save checkpoints periodically to a :ref:`SkyPilot Storage <sky-storage>`-mounted cloud bucket. For job recovery,  the program should try to reload a latest checkpoint from that path when it starts.
 
-We explain them in detail below.
+Working Task YAML
+-----------------
 
+To launch a spot job, you can simply reuse your previous task YAML.
+For example, we found the BERT fine-tuning YAML works with :code:`sky launch`, and want to
+launch it with SkyPilot managed spot jobs. 
 
-Mounting code and datasets
---------------------------------
+We can launch it with the following:
 
-To launch a spot job, users should upload their codebase and data to cloud buckets through :ref:`SkyPilot Storage <sky-storage>`.
-Note that the cloud buckets can be mounted to VMs in different regions/clouds and thus enable enables transparent job relaunching without user's intervention.
-The YAML below shows an example.
+.. code-block:: console
+
+    $ sky spot launch -n bert-qa bert_qa.yaml
 
 .. code-block:: yaml
 
-  file_mounts:
-    /code:
-      name: # NOTE: Fill in your bucket name
-      source: /path/to/your/codebase
-      persistent: true
-      mode: COPY
+  # bert_qa.yaml
+  name: bert_qa
+  
+  resources:
+    accelerators: V100:1
 
-    # If data is stored locally
-    /data:
-      name: # NOTE: Fill in your bucket name
-      source: /path/to/your/dataset
-      persistent: true
-      mode: COPY
+  # Assume your working directory is under `~/transformers`.
+  # To make this example work, please run the following command:
+  # git clone https://github.com/huggingface/transformers.git ~/transformers
+  workdir: ~/transformers
+
+  setup: |
+    # Fill in your wandb key: copy from https://wandb.ai/authorize
+    # Alternatively, you can use `--env WANDB_API_KEY=$WANDB_API_KEY`
+    # to pass the key in the command line, during `sky spot launch`.
+    echo export WANDB_API_KEY=[YOUR-WANDB-API-KEY] >> ~/.bashrc
+
+    git checkout v4.18.0
+    pip install -e .
+    cd examples/pytorch/question-answering/
+    pip install -r requirements.txt
+    pip install wandb
 
-    # If data is already on S3
-    /data2: s3://your-s3-bucket-name
+  run: |
+    cd ./examples/pytorch/question-answering/
+    python run_qa.py \
+    --model_name_or_path bert-base-uncased \
+    --dataset_name squad \
+    --do_train \
+    --do_eval \
+    --per_device_train_batch_size 12 \
+    --learning_rate 3e-5 \
+    --num_train_epochs 50 \
+    --max_seq_length 384 \
+    --doc_stride 128 \
+    --report_to wandb
 
 .. note::
 
-  Currently :ref:`workdir <sync-code-artifacts>` and :ref:`file mounts with local files <sync-code-artifacts>` are not
-  supported for spot jobs. You can convert them to :ref:`SkyPilot Storage <sky-storage>`.
+  :ref:`workdir <sync-code-artifacts>` and :ref:`file mounts with local files <sync-code-artifacts>` will be automatically translated to
```

Comment:
I feel like "translated" here may be unclear to people not familiar with the system.
How about we say: workdir and file mounts with local files are automatically uploaded to SkyPilot Storage?

Reply:
Good point! Fixed. Thanks!